pkg/config/config.go

```diff
@@ -13,6 +13,7 @@ type BoolConfig struct {
 	NoPrefixComments bool `yaml:"no-prefixComments"`
 	Debug            bool `yaml:"-"`
 	SkipGenerated    bool `yaml:"skipGenerated"`
+	CustomOrder      bool `yaml:"custom-order"`
```

Comment:
nit: `yaml:"customOrder"`. 

`NoPrefixComments` would be removed in the following release.

Reply:
done